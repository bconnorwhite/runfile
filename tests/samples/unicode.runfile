# ----------
# Unicode & International
# ----------

# Command with unicode characters
测试
  arg?  # 测试参数
  --标志: # 测试标志
  echo "Unicode command: $arg"

# Command with emoji
emoji
  🚀?   # Rocket emoji argument
  --🎯: # Target emoji flag
  echo "Emoji command: $🚀"

# Command with special unicode
special-unicode
  arg?    # Argument with ñ, é, ü, ç characters
  --flag: # Flag with special chars
  echo "Special unicode: $arg"
