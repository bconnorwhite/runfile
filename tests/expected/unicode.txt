Unicode & International
  测试            # Command with unicode characters
    arg?              # 测试参数
    --标志          # 测试标志
  emoji             # Command with emoji
    🚀?             # Rocket emoji argument
    --🎯            # Target emoji flag
  special-unicode   # Command with special unicode
    arg?              # Argument with ñ, é, ü, ç characters
    --flag            # Flag with special chars

